Logging/app.js

```diff
@@ -0,0 +1,64 @@
+const express = require('express');
+const app = express();
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const Log = require('./models/log');
+var CryptoJS = require("crypto-js");
+
+app.use(bodyParser.urlencoded({extended: false}));

```

Comment:
This is deployed on a different server right?

Reply:
yeah it is, once approved i'm going to change the URL from http://localhost:3000 in asp.net side